pom.xml

```diff
@@ -13,7 +13,7 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.509.4</version>
+		<version>1.609.1</version>
```

Comment:
Would this be a good time to change the parent to "plugin-pom"?  It's optional; you'd gain some dependencies like annotations & can specify your desired Jenkins version.